source/guides/models/repositories.md

```diff
@@ -62,6 +62,27 @@ repository.find(book.id)
   # => nil
 ```
 
+## Saving more than one record
+
+For saving butch of records you need to create `#create_many` command:
+
+```ruby
+class BookRepository
+  def create_many(list)
+    command(:create, books, result: :many).call(list)
+  end
+end
+```
+
+And after that you can use it as a `#create` method:
```

Comment:
I find this sentence misleading, it may lead to people thinking the `#create` method can now create many records instead of understanding that there's a new `#create_many` method

It also can't be used as the `#create` method, as the parameters passed are really different...

I'd rephrase to `Now you can use `#create_many` as a bulk creation method:`